primary/Tenkuu Danzai Skelter Heaven (primary).ass

```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
+Comment: 0,0:01:43.35,0:01:47.95,Signs,,0,0,0,,{\pos(342.545,73.636)}Specially Designated Public Corporation Altamira Agency{There's a typo in "Designated" (there's no 'a'), but otherwise this should be probably be left alone since SDPC is a real thing, otherwise known as a semigovernmental corporation. The initials SDPC appear later in the OP.}
+Dialogue: 0,0:01:43.35,0:01:47.95,Signs,,0,0,0,,{\pos(581.212,290.969)}The human body arms which had the program embedded\Nso that a command may be obeyed from from when born\N
+Dialogue: 0,0:01:47.85,0:01:49.36,Signs,,0,0,0,,{\pos(322.545,320.302)}AA passed many feature and years\Nand succeededin producing the first\Ngeneration at least
+Dialogue: 0,0:01:48.45,0:01:49.25,Signs,,0,0,0,,{\pos(498.4,420)}To the sake when coming
+Dialogue: 0,0:01:53.03,0:02:07.57,Signs,,0,0,0,,{\pos(69.212,197.635)}Even if they were called "Weattprhopet"{sic, from ウェットプロープ} and gave the complete same\Nprograms, become altogetherdifferent capability balance
+Dialogue: 0,0:02:07.87,0:02:29.13,Signs,,0,0,0,,{\pos(572,219.333)}She "Weattprhopet" got into "Skelter" united with each individuality, and has\Nrepeted{"repeated"} the improvement constantly in preparation for the day which should come.
 Dialogue: 0,0:03:12.20,0:03:14.57,Signs,,0,0,0,,{\pos(318.286,146.857)}Heavenly Arbiter
-Comment: 0,0:03:23.72,0:03:33.19,Signs,,0,0,0,,# News headline
-Dialogue: 0,0:03:23.72,0:03:33.19,Signs,,0,0,0,,{\pos(311.429,397.714)}Mysterious airborne entity begins moving over Tokyo airspace
-Comment: 0,0:03:33.19,0:03:38.20,Signs,,0,0,0,,# News headline top left
-Dialogue: 0,0:03:33.19,0:03:38.20,Signs,,0,0,0,,{\pos(64,178.286)}Mysterious airborne entity begins moving over Tokyo airspace
-Comment: 0,0:03:33.19,0:03:38.20,Signs,,0,0,0,,# News headline top right
-Dialogue: 0,0:03:33.19,0:03:38.20,Signs,,0,0,0,,{\pos(449.143,183.429)}Mysterious airborne entity begins moving over Tokyo airspace
-Comment: 0,0:16:54.86,0:16:59.76,Signs,,0,0,0,,# One year later
-Dialogue: 0,0:16:54.86,0:16:59.76,Signs,,0,0,0,,{\pos(325.143,254.286)}One year later...
-Comment: 0,0:17:02.37,0:17:11.98,Signs,,0,0,0,,# News headline reprise
-Dialogue: 0,0:17:02.37,0:17:11.98,Signs,,0,0,0,,{\pos(283.429,408.571)}Mysterious airborne entity appears over Tokyo once more
+Dialogue: 0,0:03:23.52,0:03:33.16,Signs,,0,0,0,,{\pos(311.429,397.714)}Mysterious Airborne Entity Begins Moving Over Tokyo Airspace
+Dialogue: 0,0:03:33.16,0:03:38.16,Signs,,0,0,0,,{\pos(64,178.286)}Mysterious Airborne Entity Begins Moving Over Tokyo Airspace
+Dialogue: 0,0:03:33.16,0:03:38.16,Signs,,0,0,0,,{\pos(449.143,183.429)}Mysterious Airborne Entity Begins Moving Over Tokyo Airspace
+Dialogue: 0,0:16:54.86,0:16:59.76,Signs,,0,0,0,,{\pos(325.143,254.286)}One Year Later
+Dialogue: 0,0:17:02.10,0:17:11.98,Signs,,0,0,0,,{\pos(283.429,408.571)}Mysterious Airborne Entity Appears over Tokyo Again
```

Comment:
I don't think you had meant to revert these lines.
```suggestion
Comment: 0,0:03:23.72,0:03:33.19,Signs,,0,0,0,,# News headline
Dialogue: 0,0:03:23.72,0:03:33.19,Signs,,0,0,0,,{\pos(311.429,397.714)}Mysterious airborne entity begins moving over Tokyo airspace
Comment: 0,0:03:33.19,0:03:38.20,Signs,,0,0,0,,# News headline top left
Dialogue: 0,0:03:33.19,0:03:38.20,Signs,,0,0,0,,{\pos(64,178.286)}Mysterious airborne entity begins moving over Tokyo airspace
Comment: 0,0:03:33.19,0:03:38.20,Signs,,0,0,0,,# News headline top right
Dialogue: 0,0:03:33.19,0:03:38.20,Signs,,0,0,0,,{\pos(449.143,183.429)}Mysterious airborne entity begins moving over Tokyo airspace
Comment: 0,0:16:54.86,0:16:59.76,Signs,,0,0,0,,# One year later
Dialogue: 0,0:16:54.86,0:16:59.76,Signs,,0,0,0,,{\pos(325.143,254.286)}One year later...
Comment: 0,0:17:02.37,0:17:11.98,Signs,,0,0,0,,# News headline reprise
Dialogue: 0,0:17:02.37,0:17:11.98,Signs,,0,0,0,,{\pos(283.429,408.571)}Mysterious airborne entity appears over Tokyo once more
```

Reply:
I used a dated file for the edit, so reverting the signs was definitely unintentional. I'll check it out for consistency and flow before the night ends.